internal/db/schema/migrations/postgres/7/02_job.up.sql

```diff
@@ -1,20 +1,28 @@
 begin;
 
-    create table job_plugin (
-        public_id wt_public_id primary key
+    create domain wt_plugin_id as text
+        not null
+        check(
+                length(trim(value)) > 10 or value = 'pi_system'
+            );
+    comment on domain wt_plugin_id is
+         '"pi_system", or random ID generated with github.com/hashicorp/vault/sdk/helper/base62';
+
+    create table plugin (
+        public_id wt_plugin_id primary key
     );
 
-    insert into job_plugin (public_id)
+    insert into plugin (public_id)
     values
-        ('jpi_internal');
+        ('pi_system');
```

Comment:
1. Should a trigger be added to prevent `pi_system` from being deleted?
2. And should `plugin_id` be an immutable column?

Reply:
I think so, for both